corehq/apps/api/tests/user_resources.py

```diff
@@ -7,8 +7,8 @@
 from flaky import flaky
 
 from corehq.apps.api.resources import v0_5
-from corehq.apps.es.tests.utils import es_test
 from corehq.apps.groups.models import Group
+from corehq.apps.es.tests.utils import es_test
```

Comment:
Yes, why not? This allows testing in ES7 and ES2 matrices.

Reply:
Sorry, my question was not clear. I was asking if commit https://github.com/dimagi/commcare-hq/pull/28270/commits/c5d65a8ff06e0b84819ea8378efa1cc5fb330500 is still needed, or if it could be reverted? That is, is there still a merge conflict with staging?